src/main/kotlin/carracing/game/domain/data/Car.kt

```diff
@@ -0,0 +1,13 @@
+package carracing.game.domain.data
+
+private const val INITIAL_POSITION = 0
+
+data class Car(
+    var position: Int = INITIAL_POSITION,
+) {
+    fun move(randomNumber: Int) {
+        if (randomNumber >= 4) {
```

Comment:
How about replacing magic numbers with named constants to make your code more readable?